src/treeview/nodes/toplevel.ts

```diff
@@ -99,7 +101,7 @@ class SubprojectNode extends BaseNode {
     benchmarks: Tests,
 
     ) {
-    super(`${parentId}-${subproject.descriptive_name}-${subproject.version}`);
+    super(`${parentId}-${subproject.descriptive_name}${subproject.version !== "undefined" ? `-${subproject.version}` : ""}`);
```

Comment:
Why are you doing it in a single line here and not in other places?

Reply:
JavaScript has a restriction that `super()` has to be the first statement in a constructor. Otherwise, I would have moved it too. Java has a similar problem.

Reply:
Ok lgtm